pkg/component/component.go

```diff
@@ -740,9 +740,9 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 
 	if err != nil {
 		// If we fail, log the output
-		log.Errorf("Unable to build files\n%v", cmdOutput)
 		s.End(false)
-		return errors.Wrap(err, "unable to execute assemble script")
+		log.Errorf("Unable to build files: %v", cmdOutput)
+		return errors.Wrapf(err, "unable to build files")
```

Comment:
We're changing the error message here. Is it really required? Is the old message incorrect? cc @kadel

Reply:
I think that either of those messages is ok.